framework/src/utils/MatrixTools.C

```diff
@@ -31,7 +31,7 @@ int inverse(const std::vector<std::vector<Real> > & m, std::vector<std::vector<R
     for (const auto & matrix_entry : rowvec)
       A.push_back(matrix_entry);
 
-  int error = inverse(A, n);
+  int error = inverse(A, (int) n);
```

Comment:
Have a look at the method.   You can see there are lots of "int"s there because of LAPACKgetrf, FORTRAN_CALL(dgetri) and LAPACK_getri.   Suggest a cleaner alternative and i'll implement it.